package.json

```diff
@@ -10,6 +10,7 @@
     }
   ],
   "devDependencies": {
+    "axios": "^0.15.3",
```

Comment:
Man, I know how to install a dependency and save it in the package.json 😂 
What I'm trying to say is: If we are compiling a static bundle, there should not be `dependencies`, only `devDependencies`. And my question is: Is there a reason why we keep dependencies and not just use `devDependencies` (including react related things)?